### ISSUE TITLE
Dynamic footer

### ISSUE THREAD

**Issue by mohamedMahdyCc** [camelcasetechsd/testing-engine#386]

**Comment by mohamedMahdyCc** [comment]
@LamiaSalah @ahmedelbougha @Ali-Galal @HeshamHassan 
Done! please confirm and close

**Comment by HeshamHassan** [comment]
**@mohamedMahdyCc** 
As agreed with **@LamiaSalah** at the meeting today..
The footer needed to be a mirror to the header. and with the same order.

**Comment by mohamedMahdyCc** [comment]
@HeshamHassan @LamiaSalah 

i discussed this with @ahmedelbougha and i think this is rejected from @Ali-Galal , so please confirm and close

**Comment by LamiaSalah** [comment]
@mohamedMahdyCc  OK, so the footer currently need to be changed same as the header, even if it's not linked to it, how would this be done?

**Comment by mohamedMahdyCc** [comment]
@LamiaSalah @HeshamHassan @ahmedelbougha @Ali-Galal 
simple documentation for skillsbuilder pro how dynamic footer works
https://docs.google.com/document/d/1uIMzRLOgoOsDfKdLei3kgRUPq52fWAXd5uBfJKbyWIs/edit?usp=sharing

**Comment by LamiaSalah** [comment]
@mohamedMahdyCC I'm not sure this is clear for me. Is this used to EDIT the current footer content?

**Comment by HeshamHassan** [comment]
**@mohamedMahdyCc** 
I tried the following scenario: 
1. Create menu item "About Us" without parent and Weight 1
2. Create menu item "The Skilled Matrix Advantage" with parent "About Us" and Weight 1
3. Create menu item "Mission And Core Values" with parent "About Us" and Weight 2
4. Create the rest of footer headers with the same method
5. The rest of footer items are not created yet.

I got the following footer:
![image](https://user-images.githubusercontent.com/11155105/51534108-e84e2680-1e55-11e9-8a03-3fbc845cd2e4.png)
